rest/blip_api_test.go

```diff
@@ -1833,7 +1833,7 @@ func TestGetRemovedDoc(t *testing.T) {
 	// Try to get rev 3 via BLIP API and assert that _removed == true
 	resultDoc, err = bt2.GetDocAtRev("foo", "3-cde")
 	assert.NoError(t, err, "Unexpected Error")
-	goassert.True(t, resultDoc.IsRemoved())
+	assert.False(t, resultDoc.IsRemoved())
```

Comment:
This difference in behaviour seems unexpected to me, or at least no longer matches any of the test description/comments. But if it is an intentional change, why is it only the BLIP response has changed, and not the REST API response below?

Reply:
As discussed, to avoid backfill in this test (and maintain the previous removal handling), move the creation of bt2 to prior to doc creation (i.e. right after we create the first blip tester).  Also add a defer bt2.close() after creation.

Reply:
Sure, moved the creation of bt2 prior to doc creation. Also add a defer bt2.close() after creation.